lib/galaxy/config/sample/job_conf.sample.yml

```diff
@@ -799,7 +799,7 @@ execution:
       function: map_tool_to_destination
       rules_module: tpv.rules
       tpv_config_files:
-        - https://raw.githubusercontent.com/galaxyproject/total-perspective-vortex/main/tests/fixtures/mapping-sample.yml
+        - https://gxy.io/tpv/db.yml
```

Comment:
This new link is pointing to https://raw.githubusercontent.com/galaxyproject/tpv-shared-database/main/tools.yml which I think makes sense. Just wanted to point out that both links are not identical.

Reply:
Thanks for pointing that out. If I recall correctly, the original link was put in just to point to a simplified example. Now that we have the shared db, we might as well point to a fixed url that can directly be copied instead?